### DISCUSSION TITLE
Regarding the Non-commercial Usage of the License

### DISCUSSION THREAD

**Discussion by BreakDroid** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#169] [Q&A] [answered]
Thank you for always using our service. We appreciate your creation of such an amazing extension.

I would like to verify the change in the license to non-commercial usage. Does this also apply to the images generated using this extension? I am asking because there is a statement indicating that the license's influence extends to derivative works.

Due to this change, some Japanese users are removing their submissions from contests. Although there is no prize money in these contests, the top-ranking works will be sold as trading cards.

Please let me know if the intention behind the license change includes prohibiting such commercial usage.

We genuinely want to continue using this incredible extension with confidence. We would be grateful if you could clearly state that the generated works are not affected by the license.

**Comment by Kahsolt** [comment]
IMHO, we'd only restrict the extension itself (code/repo) cannot be sold with a price, everyone should be free of charge to access, use, modify, redistribute it. From my personal point of view, this does NOT effect the derived works.

However, let's depend on Mr. @pkuliyi2015 to make the **final decision**.  😀

**Comment by pkuliyi2015** [comment, marked as answer]
Of course we don't restrict the outcome images. Please just do not sell the code itself, or use the code into your commercial project.

**Comment by BreakDroid** [comment]
Thank you for your response. I am sure that many wonderful creations will continue to be produced using this feature in the future.
